src/main/java/gnu/io/RXTXCommDriver.java

```diff
@@ -122,18 +121,9 @@
 		{
 			if ( ! noVersionOutput )
 			{
-				log.info("Stable Library");
-				log.info("=========================================");
-				log.info("Native lib Version = " + LibVersion );
-				log.info("Java lib Version   = " + JarVersion );
+				log.info("Stable Library Version: Native: {} Java: {}", LibVersion, JarVersion);
```

Comment:
`devel` is always false, so this is never printed. I'd just rip it out entirely. Nobody wants this to be printed automatically. If they want to log this information for their own diagnostic purposes, they can call `RXTXVersion.getVersion()`/`RXTXVersion.nativeGetVersion()` themselves.

Reply:
I think this comes from the old days when the JAR and the native libraries were shipped separately. I'd remove it, too.